onadata/libs/utils/image_tools.py

```diff
@@ -35,7 +36,8 @@ def get_dimensions((width, height), longest_side):
 
 
 def _save_thumbnails(image, path, size, suffix):
-    nm = NamedTemporaryFile(suffix='.%s' % settings.IMG_FILE_TYPE)
+    image_ext = os.path.splitext(path)[1]
```

Comment:
Can you use `image_ext = image.format`?
Or even 
`image_ext = image.format if image.format else settings.IMG_FILE_TYPE`

It avoids an extra import and extra dot in the `nm.name`.

Reply:
Ok, changed to `image.format`. I thought at first this could cause some issues, given that for `.jpg` image `image.format == "JPEG"`, but it's actually saved with proper extension.

Regarding:
`image_ext = image.format if image.format else settings.IMG_FILE_TYPE`
I'm guessing that `image.format` should never be empty or None. If it is then there are probably some other issues.

Finally, maybe `settings.IMG_FILE_TYPE` should be removed from settings, as this seems to be the only place that uses it?

Reply:
Indeed